set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     BSADBUARVDUJVEVFVGDALHJVHUNVI $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Real subtraction
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c -R $.
+
+  $( Real number subtraction. $)
+  crsub $a class -R $.
+
+  $( Based on ~ conventions-comments , I add a "(Contributed by...)" and a
+     "(Revised by...)" tag every time the proofs are equivalent.  I also add a
+     "Based on..." sentence so that any other contributors are credited.  In
+     these theorems the only difference is the use of ` -R ` instead of ` - ` .
+     All theorems are named and constructed based on the equivalent complex
+     number subtraction theorems. $)
```

Comment:
I think in the following cases, the original contributot need not be mentioned. It is sufficient to reference the original theorem.

Reply:
The whole comment can be removd.

Reply:
OK now